### PR TITLE
Adding tutorial for AKS quickstart with event grid 

### PR DESCRIPTION
**Pull request by jasonmesser7**

### CODE REVIEW

File: demoScripts/aksQuickstartEventGrid.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+* An Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/free).
+* [Azure CLI][azure-cli-install] or [Azure PowerShell][azure-powershell-install] installed.
+
+## Create an AKS cluster
+
+### [Azure CLI](#tab/azure-cli)
+
+## Define Environment Variables
+```azurecli-interactive
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:27]: Need text to describe why these are here.

File: demoScripts/aksQuickstartEventGrid.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+export RESOURCE_GROUP_NAME=myResourceGroup
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:28]: Meaningful names

File: demoScripts/aksQuickstartEventGrid.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+export RESOURCE_LOCATION=eastus
+export AKS_CLUSTER_NAME=myAKSCluster
+export NAMESPACE_NAME="myNamespace$(printf "%08d" $((RANDOM%100000000)))"
+export EVENT_GRID_HUB_NAME=myEventGridHub
+export EVENT_GRID_SUBSCRIPTION_NAME=myEventGridSubscription
+```
+
+Create an AKS cluster using the [az aks create][az-aks-create] command. The following example creates a resource group *MyResourceGroup* and a cluster named *MyAKS* with one node in the *MyResourceGroup* resource group:
+
+```azurecli-interactive
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:38]: No results block

File: demoScripts/aksQuickstartEventGrid.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION
+az aks create --resource-group $RESOURCE_GROUP_NAME --name $AKS_CLUSTER_NAME --location $RESOURCE_LOCATION  --node-count 1 --generate-ssh-keys
+```
+
+### [Azure PowerShell](#tab/azure-powershell)
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:43]: Not an Innovation Engine problem, but we should not have PowerShell and Bash intertwined like this. We should take this up with Docs team. Tabs I can cope with, as long as BASH is the default, but not inline merging like this.

File: demoScripts/aksQuickstartEventGrid.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+  "type": "Microsoft.EventHub/namespaces/eventhubs",
+  "updatedAt": "2023-02-07T23:42:23.427000+00:00"
+}
+```
+> [!NOTE]
+> The *name* of your namespace must be unique.
+
+Subscribe to the AKS events using [az eventgrid event-subscription create][az-eventgrid-event-subscription-create]:
+
+```azurecli-interactive
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:124]: No results block

File: demoScripts/aksQuickstartEventGrid.md [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+--endpoint $ENDPOINT
+```
+
+Verify your subscription to AKS events using `az eventgrid event-subscription list`:
+
+```azurecli-interactive
+az eventgrid event-subscription list --source-resource-id $SOURCE_RESOURCE_ID
+```
+
+The following example output shows you're subscribed to events from the *MyAKS* cluster and those events are delivered to the *MyEventGridHub* event hub:
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:139]: Watch hard coded names in the text, use the environment variable instead.

File: demoScripts/aksQuickstartEventGrid.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+
+When AKS events occur, you'll see those events appear in your event hub. For example, when the list of available Kubernetes versions for your clusters changes, you'll see a `Microsoft.ContainerService.NewKubernetesVersionAvailable` event. For more information on the events AKS emits, see [Azure Kubernetes Service (AKS) as an Event Grid source][aks-events].
+
+## Delete the cluster and subscriptions
+
+### [Azure CLI](#tab/azure-cli)
+
+Use the [az group delete][az-group-delete] command to remove the resource group, the AKS cluster, namespace, and event hub, and all related resources.
+
+```azurecli-interactive
```
Comment (rgardler-msft) [inline demoScripts/aksQuickstartEventGrid.md:241]: No results block

**Review by rgardler-msft — CHANGES_REQUESTED** [review]
In general all code blocks should have a corresponding results block. It's not a test document if we aren't actually testing. This doc could fail at multiple points but we wouldn't know until the very last section. This means the docs team would have to debug. The goal is to tell the docs team the precise command that failed.

Having results blocks also makes the document more easily followed by human beings as they can compare the results they get with the ones in the doc.

File: demoScripts/aksQuickstartEventGrid.md (hunk 1 above)
Comment (rgardler-msft) [reply demoScripts/aksQuickstartEventGrid.md:27]:
```suggestion
This document uses environment variables for all parameters to facilitate reuse. The default values provided here should work in most test environments. For production work you will obviously need to modify these values.

```azurecli-interactive
```

**Review by rgardler-msft — APPROVED** [review]

**Comment by rgardler-msft** [comment]
I made all the changes I recommended and tested the execution. It works fine.

I believe this can now be mereged.

**Comment by jasonmesser7** [comment]
Changes look good to me. Merging now.

**Pull request merged**